lib/fourchette/pgbackups.rb

```diff
@@ -20,16 +20,16 @@ def copy(from, to)
   private
 
   def ensure_pgbackups_is_present(heroku_app_name)
-    unless existing_backups?
+    unless existing_backups?(heroku_app_name)
       logger.info "Adding pgbackups to #{heroku_app_name}"
       @heroku.client.addon.create(heroku_app_name, { plan: 'pgbackups' })
     end
   end
 
-  def existing_backups?
-    @heroku.client.addon.list(heroku_app_name).select do |addon|
+  def existing_backups?(heroku_app_name)
+    @heroku.client.addon.list(heroku_app_name).any? do |addon|
```

Comment:
Yes, it does.
[Here](https://github.com/rainforestapp/fourchette/blob/master/lib/fourchette/heroku.rb#L98) you can see another place that `any?` is being used in same way (and also checking if any addon for the list has a specific name).

```ruby
2.1.2 :001 > [1,2,3,4,5].select { |num|  num.even?  }
 => [2, 4]
2.1.2 :002 > [1,2,3,4,5].select { |num|  num.even?  }.any?
 => true
2.1.2 :003 > [1,2,3,4,5].any? { |num|  num.even?  }
 => true
```